### PR TITLE
Use osadl-matrix library

### PR DESCRIPTION
**Pull request by priv-kweihmann**

### CODE REVIEW

File: tests/test_compat_matrix.py [hunk 1]
```diff
@@ -18,12 +18,10 @@
 
 from flict.flictlib.compat_matrix import CompatibilityMatrix, CompatMatrixStatus
 from flict.var import VAR_DIR
-
-MATRIX_FILE=os.path.join(VAR_DIR, "osadl-matrix.csv")
-
+from osadl_matrix import OSADL_MATRIX
```
Comment (jens-erdmann) [inline tests/test_compat_matrix.py:21]: I am surprised that flake8 is happy with this. I would have expected more newlines here.

File: tests/test_report.py [hunk 2]
```diff
@@ -21,15 +21,15 @@
 
 TRANSLATION_FILE   = os.path.join(VAR_DIR, "translation.json")
 RELICENSE_FILE     = os.path.join(VAR_DIR, "relicense.json")
-MATRIX_FILE = os.path.join(VAR_DIR, "osadl-matrix.csv")
+from osadl_matrix import OSADL_MATRIX
```
Comment (jens-erdmann) [inline tests/test_report.py:24]: Same here.

**Review by jens-erdmann — APPROVED** [review]
Rest looks fine to me.

File: tests/test_compat_matrix.py (hunk 1 above)
Comment (priv-kweihmann) [reply tests/test_compat_matrix.py:21]: Well, I'm not sure if we might have this newline check disabled, but certainly I don't see the need to enforce it for the test-files... :thinking:... I think we don't enforce any flake8 findings for the files under tests...

**Pull request merged**